.github/workflows/_kotlin.yml

```diff
@@ -78,3 +75,47 @@ jobs:
         run: |
           echo -n "$FIREBASE_APP_DISTRIBUTION_CREDENTIALS" > $FIREBASE_CREDENTIALS_PATH
           ./gradlew --info appDistributionUploadRelease uploadCrashlyticsSymbolFileRelease
+
+  ui-test:
+    name: ui-test-api-${{ matrix.api-level }}
```

Comment:
Calling it a `ui-test` because that has specific meaning in an android context. See: https://developer.android.com/training/testing/instrumented-tests/ui-tests

Other UI tests can base it off of this :)